### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by Samuellekpor**
In this feature branch, I did the following:

1. Create a new rails app.
2. Set up a Postgres database.
3. Create the necessary migration files.
4. Run the migration file to build the project schema.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 20_230_202_002_529) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension 'plpgsql'
+
+  create_table 'comments', force: :cascade do |t|
+    t.text 'text'
+    t.date 'updatedAt'
+    t.date 'createdAt'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly run new migration and remove these columns from the comment table. We already have `created_at` and `updated_at` as generated by Rails.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    t.datetime 'updated_at', null: false
+    t.bigint 'author_id', null: false
+    t.bigint 'post_id', null: false
+    t.index ['author_id'], name: 'index_comments_on_author_id'
+    t.index ['post_id'], name: 'index_comments_on_post_id'
+  end
+
+  create_table 'likes', force: :cascade do |t|
+    t.date 'updatedAt'
+    t.date 'createdAt'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly run new migration and remove these columns from the likes table. We already have `created_at` and `updated_at` as generated by Rails.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    t.index ['post_id'], name: 'index_likes_on_post_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'text'
+    t.date 'updatedAt'
+    t.date 'createdAt'
+    t.integer 'commentsCounter'
+    t.integer 'likesCounter'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly run new migration and remove these columns from the post table. We already have `created_at` and `updated_at` as generated by Rails.

- [ ] Kindly rename these columns from `commentsCounter` and `likesCounter` to `comments_counter` and `likes_counter` based on Ruby conventions for column names. We use snake casing for variables and columns, not camel case

File: db/schema.rb [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    t.index ['author_id'], name: 'index_posts_on_author_id'
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.string 'photo'
+    t.text 'bio'
+    t.date 'updatedAt'
+    t.date 'createdAt'
+    t.integer 'postsCounter'
```
Comment (TedLivist) [inline db/schema.rb]: - [ ] Kindly run new migration and remove these columns from the user table. We already have `created_at` and `updated_at` as generated by Rails.

- [ ] Kindly run a new migration and rename this column, `postsCounter`, to `posts_counter`

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Samuellekpor,

Good job so far!
✔️ Use of PostgreSQL for DB
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lucas-Masaba — APPROVED** [review]
# Hello @Samuellekpor  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

almost-there

## Highlights 🍾 
- Changes were made as requested by the previous reviewer 👍 
- Linter checks passing ✔️ 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**